document-store/src/main/java/org/hypertrace/core/documentstore/OrderBy.java

```diff
@@ -26,4 +27,30 @@ public boolean isAsc() {
   public void setIsAsc(boolean isAsc) {
     this.isAsc = isAsc;
   }
+
+  @Override
+  public String toString() {
+    return "OrderBy{" +
+        "field='" + field + '\'' +
+        ", isAsc=" + isAsc +
+        '}';
+  }
+
+  @Override
+  public boolean equals(Object o) {
```

Comment:
No unit tests for this?

Reply:
`OrderBy` doesn't have any explicit unit tests. It is mostly tested as part of `QueryTest`. Did the same thing for its `equals` method.